pkg/web/templates/send_message.html

```diff
@@ -15,26 +15,26 @@ <h1 class="mx-auto font-semibold text-2xl md:text-3xl text-center text-balance">
       You will need a valid email address for your counterparty. If your counterparty is in the TRISA or TRP networks,
       <a href="/send-envelope" class="underline text-blue-700">send a secure envelope</a> instead.
     </p>
-    <form id="sunrise-msg-form" hx-post="" hx-target="#preview_msg" hx-indicator="#loader" hx-ext="json-enc" method="post">
+    <form id="sunrise-msg-form" hx-post="/sunrise/message" hx-target="#sunrise" hx-swap="outerHTML" hx-indicator="#loader" hx-ext="json-enc" method="post">
```

Comment:
I went with the implemented route for now. It currently returns a `405` error. If this needs to change, let me know.